lib/v3-lab.js

```diff
@@ -93,7 +93,9 @@ window.MathJax = {
             'input/tex-full',
             'input/mml',
             'output/chtml',
-            'ui/menu'
+          'ui/menu',
+          '[tex]/textmacros',
+          '[tex]/textcomp'
```

Comment:
You are right. that was not intended to be in the PR at all.

Reply:
Just FYI, I have made a PR that updates the lab so that you can select textmacros packages just as you can TeX packages.  (Plus lots of other clean up and new features.)